PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return (Convert.ToDouble(numberCorrectAnswers) / Convert.ToDouble(numberOfQuestions) * 100) + "%";
```

Comment:
Could have just done a cast as well, but running a convert isn't bad either. Only change I would have made would have been to call Math.Round in case the result wasn't a whole number after doing the division
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```